webhook_server_container/libs/github_api.py

```diff
@@ -759,6 +759,41 @@ def close_issue_for_merged_or_closed_pr(self, hook_action):
                 issue.edit(state="closed")
                 break
 
+    @ignore_exceptions(logger=FLASK_APP.logger)
+    def delete_remote_tag_for_merged_or_closed_pr(self, hook_action):
+        pr_tag = f"pr-{self.pull_request.number}"
+        # run regctl as a container:
+        base_regctl_command = (
+            "podman run --rm --net host  -v regctl-conf:/home/appuser/.regctl/ ghcr.io/regclient/regctl:latest"
+        )
+        registry_info = self.container_repository.split("/")
+        registry_url = "" if len(registry_info) < 3 else registry_info[0]
+        # First we need to execute regctl login command before we can delete the tag:
+        rc, _, _ = run_command(
+            command=f"{base_regctl_command} registry login {registry_url} -u {self.container_repository_username} "
+            f"-p {self.container_repository_password}",
+            verify_stderr=True,
```

Comment:
Do not `verify_stderr` since the command have warnings when running it.

Reply:
ok.